src/fosslight_yocto/create_oss_report_for_yocto.py

```
@@ -922,26 +934,39 @@ def create_dir(dir_name):
+            if isinstance(df2, pd.DataFrame) and not df2.empty:
+                existing_names.update(df2["OSS_NICKNAME"].tolist())
+
+        for link_chunk in chunker(links, 500):
+            if not link_chunk: continue
+            link_str = ','.join([f"'{pymysql.converters.escape_string(l)}'" for l in link_chunk])
+
+            q3 = f"SELECT DISTINCT DOWNLOAD_LOCATION FROM OSS_DOWNLOADLOCATION WHERE DOWNLOAD_LOCATION IN ({link_str})"
+            df3 = get_list_by_using_query(db_cur, q3, ["DOWNLOAD_LOCATION"])
+            if isinstance(df3, pd.DataFrame) and not df3.empty:
+                existing_links.update(df3["DOWNLOAD_LOCATION"].tolist())
+    except Exception as ex:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)